RELEASE/scripts/autoscend/auto_sim.ash

```diff
@@ -47,7 +47,7 @@ void printSimSuggested()
 	formattedSimPrint(maxedPoolSkill, "Pool Shark", "Lucky! adv which permanently increases your pool skill. It is possible for Mafia to not realize you have maxed this. If you are confident you have, enter the following in the CLI: `set poolSharkCount=25`");
 
 	// if we have combat locket, check if we have used monsters in there
-	if(auto_haveCombatLoversLocket())
+	if(auto_canReminisceWithLocket())
 	{
```

Comment:
Possession of the locket and ability to equip it and benefit from it's effect is distinct from being able to use it to reminisce in G-Lover. Since it isn't doesn't require hitting the inv_use url reminiscing is still possible in G-Lover.